base/linked_list.jl

```diff
@@ -1,7 +1,15 @@
 # This file is a part of Julia. License is MIT: https://julialang.org/license
 
+
+"""
+    Base.IntrusiveLinkedList{T}
+
+Is used like this Base.IntrusiveLinkedList{Base.LinkedListItem{T}} as LinkedList{T}.
```

Comment:
```suggestion
Is used like this `Base.IntrusiveLinkedList{Base.LinkedListItem{T}}` as `LinkedList{T}`.
```
This sentence seems incomplete? Is there supposed to be a colon after "this", i.e., is the text afterwards meant to tbe a code example? But then what is the "as LinkedList{T}" about?